src/main/java/uk/gov/ons/ctp/integration/rhsvc/repository/RespondentCaseRepository.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.ons.ctp.integration.rhsvc.repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.ctp.common.cloud.RetryableCloudDataStore;
+import uk.gov.ons.ctp.common.error.CTPException;
+import uk.gov.ons.ctp.common.event.model.CaseUpdate;
+
+/** A Repository implementation for CRUD operations on Case data entities */
+@Service
+public class RespondentCaseRepository {
```

Comment:
All these repository names are a bit wordy. Shame you didn't just call this `CaseRepository`  and likewise with the others.

Reply:
I agree

Reply:
Have renamed. Also associated test cases.